examples/js/loaders/OBJLoader.js

```diff
@@ -376,18 +517,27 @@ THREE.OBJLoader.prototype = {
 
 				material = this.materials.create( object.material.name );
 
+				// mtl etc. loaders probably can't create line materials correctly, copy properties to a line material.
+				if (isLine && material && !(material instanceof THREE.LineBasicMaterial)) {
+
+					var materialLine = new THREE.LineBasicMaterial();
+					materialLine.copy(material);
+					material = materialLine;
+
+				}
+
 			}
 
 			if ( !material ) {
 
-				material = new THREE.MeshPhongMaterial();
+				material = ( !isLine ? new THREE.MeshPhongMaterial() : new THREE.LineBasicMaterial() );
 				material.name = object.material.name;
 
 			}
 
 			material.shading = object.material.smooth ? THREE.SmoothShading : THREE.FlatShading;
```

Comment:
I like the idea of moving all of this material assignment stuff into a `.apply()` function somewhere, but special cases like the smoothing here might force us have `OBJLoader` use the `MTLLoader` automatically during parsing, or right before thisp